### PR TITLE
Allow passing Daily token payload into `generateToken()`

### PR DESCRIPTION
**Pull request by lazeratops**
This PR allows the caller to pass a `DailyPayload` object to the OT `generateToken()` function. Everything in the explicitly passed Daily payload takes priority over other options.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,28 +11,30 @@ npm install daily-opentok-node
 When requiring the library in your project, change the following line:
 
 ```typescript
-const OpenTok = require('opentok');
+const OpenTok = require("opentok");
```
Comment (lazeratops) [inline README.md:14]: Autofixed by prettier

**Review by jamsea — APPROVED** [review]
This looks good to me what do you think @harshithpabbati

**Comment by harshithpabbati** [comment]
> This looks good to me what do you think @harshithpabbati 

Ah, I didn't press the submit button lol

**Review by harshithpabbati — APPROVED** [review]
LGTM!

**Pull request merged**